components/contribution-flow/ContributionFlowHeader.js

```diff
@@ -27,14 +29,20 @@ class NewContributionFlowHeader extends React.Component {
         ),
       }),
     }).isRequired,
+    loggedInAccount: PropTypes.object,
     LoggedInUser: PropTypes.object,
     intl: PropTypes.object,
     isEmbed: PropTypes.bool,
   };
 
   render() {
-    const { collective, isEmbed } = this.props;
+    const { collective, isEmbed, loggedInAccount } = this.props;
     const contributors = collective.contributors?.nodes;
+    let loggedInUserTotalDonations = 0;
+
+    if (loggedInAccount?.memberOf?.nodes?.length > 0) {
+      loggedInUserTotalDonations = loggedInAccount.memberOf.nodes[0].totalDonations.value * 100;
```

Comment:
We should take all memberships into account. Your total donated is the sum of all `memberOf.nodes.totalDonations.valueInCents`.

Reply:
You're right, we're only looking at the `FromCollectiveId`<>`CollectiveId` there. All good!